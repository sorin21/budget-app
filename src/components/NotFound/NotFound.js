import React, { Component } from 'react';

class NotFound extends Component {
  render() {
    return (
      <div>
        404 error! Page not Found
      </div>
    );
  }
}

export default NotFound;