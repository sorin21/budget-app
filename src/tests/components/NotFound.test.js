import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../../components/NotFound/NotFound';

test('should render NotFound component', () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});