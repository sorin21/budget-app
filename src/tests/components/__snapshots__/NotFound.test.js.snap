// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound component 1`] = `
<div>
  404 error! Page not Found
</div>
`;
